lnwallet/channel.go

```diff
@@ -7272,10 +7272,10 @@ func newOutgoingHtlcResolution(signer input.Signer,
 				ChanType:       chanType,
 				ShortChanID:    chanState.ShortChanID(),
 				Initiator:      chanState.IsInitiator,
-				CommitBlob:     chanState.RemoteCommitment.CustomBlob, //nolint:lll
+				CommitBlob:     chanState.LocalCommitment.CustomBlob, //nolint:lll
```

Comment:
nit: this looks like a fix to previously added code in this PR. Should this be squashed with the commit that introduced the code?

Reply:
SGTM, have a few other instances where I need to squash bug fixes into existing commits.